src/platform/PICO/platform_mcu.h

```diff
@@ -21,26 +21,45 @@
 
 #pragma once
 
-#if defined(RP2350)
+#define _ADDRESSMAP_H
+
+#define NVIC_PriorityGroup_2         0x500
+
+#define SPI_IO_AF_CFG           0
+#define SPI_IO_AF_SCK_CFG_HIGH  0
+#define SPI_IO_AF_SCK_CFG_LOW   0
+#define SPI_IO_AF_SDI_CFG       0
+#define SPI_IO_CS_CFG           0
+
+
+
+// Register address offsets for atomic RMW aliases
+#define REG_ALIAS_RW_BITS  (_u(0x0) << _u(12))
+#define REG_ALIAS_XOR_BITS (_u(0x1) << _u(12))
+#define REG_ALIAS_SET_BITS (_u(0x2) << _u(12))
+#define REG_ALIAS_CLR_BITS (_u(0x3) << _u(12))
 
 #include "RP2350.h"
+#include "pico/stdlib.h"
+
+#if defined(RP2350A) || defined(RP2350B)
 
 typedef enum {DISABLE = 0, ENABLE = !DISABLE} FunctionalState;
 
-#define I2C_TypeDef          I2C0_Type
+#define I2C_TypeDef          i2c0_hw
 //#define I2C_HandleTypeDef
-#define GPIO_TypeDef         IO_BANK0_Type
+#define GPIO_TypeDef         io_bank0_hw
```

Comment:
`io_bank0_hw_t` otherwise it results in

```
./lib/main/pico-sdk/rp2350/hardware_structs/include/hardware/structs/io_bank0.h:448:23: error: expected identifier or '(' before 'io_bank0_hw_t'
  448 | #define io_bank0_hw ((io_bank0_hw_t *)IO_BANK0_BASE)
      |                       ^~~~~~~~~~~~~
./src/platform/PICO/platform_mcu.h:52:30: note: in expansion of macro 'io_bank0_hw'
   52 | #define GPIO_TypeDef         io_bank0_hw
      |                              ^~~~~~~~~~~
./src/main/drivers/io.h:65:1: note: in expansion of macro 'GPIO_TypeDef'
   65 | GPIO_TypeDef* IO_GPIO(IO_t io);
```